src/createServer.js

```diff
@@ -1,8 +1,51 @@
+/* eslint-disable no-console */
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+const PORT = require('./PORT');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((request, response) => {
+    response.setHeader('Content-type', 'text/plain');
+
+    if (request.url === '/favicon.ico') {
+      return;
+    }
+
+    if (request.url.includes('..')) {
```

Comment:
We shouldn't have separate condition for this case. All we need to do is to create a path with hardcoded beginning + file name, for example: 
```javascript
fs.readFile(`./public/${fileName}`)
```
The `./public` part is fixed. The only dynamic part is the file name.